### ISSUE TITLE
Add yandex and saucenao to image search list pls

### ISSUE THREAD

**Issue by Amber318** [rr-/szurubooru#420]
/thread
![image](https://user-images.githubusercontent.com/87118974/124889128-69c2b080-dfdf-11eb-8b13-810ace9fd931.png)

**Comment by neobooru** [comment]
I don't think that these providers will be added, as most maintainers are against adding 'niche'/nsfw search providers (#367, #374). It'd probably get pretty cluttered too after a while.
A solution which has been proposed a few times before would be to make these links configurable, which I think would be a great solution. If something like that is implemented then you could change these links yourself. Would this work for you?

**Comment by fsnowdin** [comment]
Hi, I'm hosting the actual booru Amber318 pictured. Regarding your configurable links proposal: Yup! I like the change and would love to see it.

**Comment by neobooru** [comment]
> Hi, I'm hosting the actual booru Amber318 pictured. Regarding your configurable links proposal: Yup! I like the change and would love to see it.

You can also simply edit `client\html\post_readonly_sidebar.tpl` lines 58-60 to edit the current selection of search providers. Not the cleanest solution, but I guess it would work for the moment.

**Comment by fsnowdin** [comment]
> You can also simply edit client\html\post_readonly_sidebar.tpl lines 58-60 to edit the current selection of search providers. Not the cleanest solution, but I guess it would work for the moment.

Thanks so much for the instructions. Aside from getting the search query URLs right, it was super simple to just add another search provider.
 
Here's the 2 lines of HTML I added to get Yandex and SauceNAO working in `client/html/post_readonly_sidebar.tpl`:
``` html
<a href='https://yandex.com/images/search?rpt=imageview&url=<%- encodeURIComponent(ctx.post.fullContentUrl) %>'>Yandex</a> &middot;
<a href='https://saucenao.com/search.php?url=<%- encodeURIComponent(ctx.post.fullContentUrl) %>'>SauceNAO</a>
```
